src/main/java/mods/railcraft/common/carts/EntityTunnelBore.java

```diff
@@ -857,6 +865,7 @@ protected float getLayerHardness(BlockPos targetPos, BlockRailBase.EnumRailDirec
             IBoreHead head = (IBoreHead) boreSlot.getItem();
             float dig = 2f - head.getDigModifier();
             hardness *= dig;
+            hardness /= (1 + EnchantmentHelper.getEnchantmentLevel(Enchantments.EFFICIENCY, boreSlot));
```

Comment:
pickaxe shovel and axe I think, the code is then there somewhere with the tool types it looks at